app/web_hub.go

```diff
@@ -687,6 +687,7 @@ func (i *hubConnectionIndex) RemoveInactiveConnections() {
 	now := model.GetMillis()
 	for conn := range i.byConnection {
 		if !conn.active && now-conn.lastUserActivityAt > i.staleThreshold.Milliseconds() {
+			conn.App.Srv().userService.ReturnSessionToPool(conn.GetSession())
```

Comment:
What's the reason for returning the session in here vs doing it inside `hubConnectionIndex.Remove()`? Is it possible we could leak connections this way?

Reply:
My bad, I'll change it.